Xamarin.Forms.Material.iOS/MaterialEditorRenderer.cs

```diff
@@ -0,0 +1,116 @@
+﻿using CoreGraphics;
+using UIKit;
+using MaterialComponents;
+using System.Threading.Tasks;
+using Foundation;
+using System;
+
+namespace Xamarin.Forms.Platform.iOS.Material
+{
+	public class MaterialEditorRenderer : EditorRendererBase<MaterialMultilineTextField>, IMaterialEntryRenderer
+	{
+		bool _hackHasRan = false;
+
+		protected override MaterialMultilineTextField CreateNativeControl()
+		{			
+			return new MaterialMultilineTextField(this, Element);
+		}
+
+		protected override void SetBackgroundColor(Color color)
+		{
+			ApplyTheme();			
+		}
+
+
+		protected internal override void UpdateTextColor()
+		{
+			Control?.UpdateTextColor(this);
+		}
+
+
+		protected virtual void ApplyTheme()
+		{
+			Control?.ApplyTheme(this);
+		}
+
+		protected internal override void UpdateFont()
+		{
+			base.UpdateFont();
+			Control?.ApplyTypographyScheme(Element);
+		}	
+
+		protected internal override void UpdatePlaceholderText()
+		{
+			if (Control == null || !_hackHasRan)
+				return;
+
+			Control.UpdatePlaceholder(this);
+		}
+
+		protected internal override void UpdatePlaceholderColor()
+		{
+			if (Control == null || !_hackHasRan)
+				return;
+
+			Control.UpdatePlaceholder(this);
+		}
+
+		protected internal override void UpdateText()
+		{
+			if (!_hackHasRan)
+				return;
+
+			base.UpdateText();
+		}
+
+		protected override void OnElementChanged(ElementChangedEventArgs<Editor> e)
+		{
+			base.OnElementChanged(e);
+			InitialPlaceholderSetupHack();
+		}
+
+		protected internal override void UpdateAutoSizeOption()
+		{
+			base.UpdateAutoSizeOption();
+			Control.ExpandsOnOverflow = Element.AutoSize == EditorAutoSizeOption.TextChanges;
+		}
+
+
+		// this is required to force the placeholder to size correctly if it starts out prefilled
+		void InitialPlaceholderSetupHack()
```

Comment:
If you set the placeholder and text of the control before focused the placeholder floats to the top left but doesn't resize. The following code was the only way I could figure out how to make it work. Once it's fixed upstream we can remove this code.

Reply:
Has this been fixed? We are a few versions behind.

Reply:
Maybe? I found this https://github.com/material-components/material-components-ios/commit/99871be6cd2ca9b365d99d46a6c28a93ac22e802

Once we get 3.6.0 out we can get updated to latest material and then go from there